react-app/src/app/components/Board/index.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-use-before-define */
+import React, { Component } from 'react';
```

Comment:
`import React, { Component, Fragment } from 'react';`

And use Fragment directly